lib/flight_job/models/script.rb

```diff
@@ -114,19 +108,15 @@ def exists?
     end
 
     def notes
-      @notes ||= if File.exist? notes_path
-                   File.read notes_path
-                 else
-                   ''
-                 end
+      @notes ||= Notes.new(id).tap { |n| n.read }
     end
 
-    def script_path
-      @script_path ||= File.join(FlightJob.config.scripts_dir, id, 'script.sh')
+    def initialize_notes(notes = "")
+      @notes ||= Notes.new(id).tap { |n| n.write(notes) }
```

Comment:
I'm not keen on needing an explicit call to `write` here.  I think `Notes.new(id, notes)` would be better.  Where the `notes` parameter is optional.